### PR TITLE
build: Use Xcode 6.4 [7.0]

### PR DESCRIPTION
**Pull request by peter-b**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -24,8 +24,8 @@ WINE ?= wine
 
 # Some magic to control which versions of iOS we try to build.  N.b. you may
 # also need to modify the buildbot configuration
-IPHONEOS_VERSIONS ?= 8.2 8.3
-IPHONESIMULATOR_VERSIONS ?= 5.1 6.1 7.1 8.2 8.3
+IPHONEOS_VERSIONS ?= 8.2 8.3 8.4
```
Comment (livecodefraser) [inline Makefile]: You can remove 8.3 from this list

**Comment by livecodefraser** [comment]
@livecode-vulcan review okay 3acf827

**Comment by livecodefraser** [comment]
@livecode-vulcan review okay 1b1c402

**Comment by livecode-vulcan** [comment]
💙 review by @livecodefraser ok 1b1c40257062b17244dd1f86c3669446c071a1c2

**Comment by livecode-vulcan** [comment]
😎 test success 1b1c40257062b17244dd1f86c3669446c071a1c2
- try-linux-x86: [success](https://vulcan.livecode.com/builders/try-linux-x86/builds/349)
- try-win-x86: [success](https://vulcan.livecode.com/builders/try-win-x86/builds/412)
- try-ios: [success](https://vulcan.livecode.com/builders/try-ios/builds/294)
- try-android-armv6: [success](https://vulcan.livecode.com/builders/try-android-armv6/builds/360)
- try-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-linux-x86_64/builds/295)
- try-mac: [success](https://vulcan.livecode.com/builders/try-mac/builds/286)

**Comment by peter-b** [comment]
@livecode-vulcan retry 1b1c402

**Comment by livecode-vulcan** [comment]
😞 test failure 1b1c40257062b17244dd1f86c3669446c071a1c2
- try-linux-x86: [failure](https://vulcan.livecode.com/builders/try-linux-x86/builds/370)
- try-win-x86: [pending](https://vulcan.livecode.com/builders/try-win-x86/builds/433)
- try-ios: [failure](https://vulcan.livecode.com/builders/try-ios/builds/315)
- try-android-armv6: [pending](https://vulcan.livecode.com/builders/try-android-armv6/builds/381)
- try-linux-x86_64: [failure](https://vulcan.livecode.com/builders/try-linux-x86_64/builds/316)
- try-mac: [failure](https://vulcan.livecode.com/builders/try-mac/builds/307)

**Comment by livecodefraser** [comment]
@livecode-vulcan review okay 5619afc

**Comment by livecode-vulcan** [comment]
💙 review by @livecodefraser ok 5619afc1ae9632bba122b276fd46eac8b7abdb72

**Comment by livecode-vulcan** [comment]
😞 test failure 5619afc1ae9632bba122b276fd46eac8b7abdb72
- try-linux-x86: [success](https://vulcan.livecode.com/builders/try-linux-x86/builds/374)
- try-win-x86: [failure](https://vulcan.livecode.com/builders/try-win-x86/builds/437)
- try-ios: [success](https://vulcan.livecode.com/builders/try-ios/builds/320)
- try-android-armv6: [success](https://vulcan.livecode.com/builders/try-android-armv6/builds/385)
- try-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-linux-x86_64/builds/320)
- try-mac: [success](https://vulcan.livecode.com/builders/try-mac/builds/311)

**Pull request merged**